### PR TITLE
Fix up 1.9.0 migration for new parentid

### PR DESCRIPTION
**Pull request by denis-yuen**
We might also be generating openapi slightly differently in develop?

### CODE REVIEW

**Comment by denis-yuen** [comment]
Why no travis-ci?

**Comment by denis-yuen** [comment]
```
ERROR [2020-03-25 19:39:04,918] org.apache.tomcat.jdbc.pool.ConnectionPool: Unable to create initial connections of pool.

! org.postgresql.util.PSQLException: FATAL: remaining connection slots are reserved for non-replication superuser connections

! at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
```
boo

**Comment by denis-yuen** [comment]
Three times, sigh.

File: dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java [hunk 1]
```diff
@@ -97,10 +97,14 @@ public void testWorkflowMigration() throws Exception {
         // Refresh should work
         workflow = workflowApi.refresh(workflow.getId());
         assertEquals("Workflow should be FULL mode", Workflow.ModeEnum.FULL, workflow.getMode());
-        assertTrue("All versions should be legacy", workflow.getWorkflowVersions().stream().allMatch(workflowVersion -> workflowVersion.isLegacyVersion()));
+        assertTrue("All versions should be legacy", workflow.getWorkflowVersions().stream().allMatch(WorkflowVersion::isLegacyVersion));
+
+        // seems like handleGitHubRelease has issue with versions that already exist
+        testingPostgres.runUpdateStatement("delete from workflowversion where parentid = " + workflow.getId());
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java]: FYI @agduncan94  looks like handleGitHubRelease might be creating duplicates

Comment (coverbeck) [reply dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java]: Andrew and I were looking at this and I'm trying to fix.

**Pull request merged**